pinot-plugins/pinot-file-system/pinot-gcs/src/main/java/org/apache/pinot/plugin/filesystem/GcsPinotFS.java

```diff
@@ -191,6 +191,9 @@ public boolean mkdir(URI uri) throws IOException {
       if (path.equals(DELIMITER)) {
         return true;
       }
+      if (isDirectory(uri)) {
```

Comment:
move this to line 189?

Reply:
Wanted to return fast if path is just the root. This way it minimizes gcs calls. What do you think?

Reply:
got it .